docs/README.md

```diff
@@ -1,21 +1,21 @@
 ## documentation
-[Manual scans - AWS](../manual_scans/aws/README.md)  
```

Comment:
the double spaces at the end of each line force a new line when the README is viewed in the browser - so they need to stay and be excluded from pre-commit

Reply:
excluded